vaccine_feed_ingest/runners/me/maine_gov/normalize.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import re
+import sys
+
+from vaccine_feed_ingest.schema import schema  # noqa: E402
+
+CITY_RE = re.compile(r"^([\w ]+), NY$")
+# the providerName field smells like it's being parsed from someplace else,
+# a good number of them have leading \u1d42 and/or *, which we want to clean.
+# there's a bunch with a city name in them, but no real pattern to it, so
+# we'll leave that for now.
+NAME_CLEAN_RE = re.compile("^[\u1d42*]+")
+
+logger = logging.getLogger(__name__)
+
+
+def _get_name(site: dict) -> str:
+    return site["providerName"]
+
+
+def _get_city(site: dict) -> str:
+    return site["city"]
+
+
+def _get_id(site: dict) -> str:
+    name = _get_name(site)
+    city = _get_city(site)
+    return f"me:maine_gov:{name}:{city}"
+
+
+def _get_source(site: dict, timestamp: str) -> schema.Source:
+    return schema.Source(
+        data=site,
+        fetched_at=timestamp,
+        fetched_from_uri="https://www.maine.gov/covid19/vaccines/vaccination-sites",
+        id=_get_id(site),
+        source="me:maine_gov",
+    )
+
+
+def _get_contacts(site: dict):
+    ret = []
+    for phone_number in site["phoneNumber"]:
+        ret.append(schema.Contact(phone=phone_number))
+    for website in site["website"]:
+        ret.append(schema.Contact(website=website))
+        ret.append(schema.Contact(other=site["schedulingInfo"]))
+    return ret
+
+
+def _get_notes(site: dict):
+    ret = []
+    ret.append("city:" + site["city"])
```

Comment:
@eliblock I wonder how we can include this data in the schema without resorting to notes. These Maine sites have city & county info but not zip/street1 so they don't fit into either of the location objects right now.

Reply:
I _think_ the plan is to have a data-cleaning process add this info later in the pipeline if it's not available. @bryanculbertson - could you share some guidance here?

Reply:
Do we have a resolution here?

If not, do we think this issue should be blocking?

Reply:
Not yet, I think. Definitely shouldn't block—we can patch it up in a future PR if the schema changes to allow.